eclair-core/src/main/scala/fr/acinq/eclair/payment/PaymentEvents.scala

```diff
@@ -230,4 +230,6 @@ object PaymentFailure {
     }))
   }
 
-}
\ No newline at end of file
+}
+
+case class ExperimentMetrics(amount: MilliSatoshi, fees: MilliSatoshi, success: Boolean, duration: Long, timestamp: Long, experimentName: String)
```

Comment:
The implicit assumption is that experiments are immutable, since we only persist the experiment name.

The name is too generic though, how about:

```suggestion
case class PathFindingExperimentMetrics(amount: MilliSatoshi, fees: MilliSatoshi, success: Boolean, duration: Long, timestamp: Long, experimentName: String)
```